client/extensions/woocommerce/index.js

```diff
@@ -178,3 +183,6 @@ export default function() {
 		page( '/store/stats/:type/:unit/:site', siteSelection, navigation, StatsController );
 	}
 }
+
+initExtension();
```

Comment:
I find this a bit inconsistent with how we're automatically loading extension reducers. Could/Should we have `extensions-loader` take care of data-layer middleware, too?